app/controllers/webhooks_controller.rb

```diff
@@ -0,0 +1,27 @@
+class WebhooksController < ApplicationController
```

Comment:
Including it in apps already defining a `WebhooksController` could lead to interesting results